package.yaml

```diff
@@ -68,6 +68,9 @@ library:
     - deriving-compat           >= 0.3 && < 0.5
     - directory
     - hashable
+    - http-types
+    - http-client
+    - http-client-tls
```

Comment:
So, now that package.yaml is updated, which is correct, we also need to run hpack and commit the .cabal file so that people’s nix-shell will see it and for the Travis build.

Reply:
now i did it. thanks.